app/repositories/sessions_repository.rb

```diff
@@ -6,4 +6,29 @@ def find_with_streams(uuid:)
   def active_in_last_7_days
     Session.where('last_measurement_at > ?', Time.current - 7.days)
   end
+
+  def fixed_active_government_sessions(data)
```

Comment:
So far we were trying to expose a method interface, so it would be clear what params it takes.
I see that we have a long list here 🤔